modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -60,17 +60,22 @@ class Configuration(object):
         """Password for HTTP basic authentication
         """
 {{#hasOAuthMethods}}
-        self.access_token = ""
+        self.access_token = None
         """access token for OAuth/Bearer
         """
 {{/hasOAuthMethods}}
 {{^hasOAuthMethods}}
 {{#hasBearerMethods}}
-        self.access_token = ""
+        self.access_token = None
         """access token for OAuth/Bearer
         """
 {{/hasBearerMethods}}
 {{/hasOAuthMethods}}
+        self.security_schemes = None
```

Comment:
Why do we need to add this parameter?
Why not just iterating over the keys of self.auth_settings()?

Reply:
You are right, initially I thought this would be needed, I will remove it.